keyboards/bobpad/readme.md

```diff
@@ -0,0 +1,19 @@
+# bobpad
+
+![bobpad](https://i.imgur.com/93zguh1.jpg)
+
+A bentopad based macropad with incline
+
+* Keyboard Maintainer: [Ananya Kirti](https://github.com/AnanyaKirti)
+* Hardware Supported: Custom 3D printed Case
+* Hardware Availability: Contact bob at IMKC Discord
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
added how to enter boot loader! thanks!